InventoryInStorePickupFrontend/view/frontend/web/js/model/pickup-locations-service.js

```diff
@@ -50,7 +52,10 @@ define([
 
             return storage
                 .get(serviceUrl, {}, false)
-                .then(function (address) {
+                .then(function (result) {
```

Comment:
This is not a part of the hotfix, it is a fix for this issue: https://github.com/magento/inventory/issues/3209
Commit: https://github.com/magento-sparta/inventory/commit/b7e39669cb112432bc98cb0aca08431adc509773

I guess it was added by accident

Reply:
Fixed. Thank you!